README.md

```diff
@@ -65,10 +65,12 @@ The following commands are useful to open and close folds:
 - `zR`: opens all folds
 - `zm`: increases fold level throughout the buffer
 - `zM`: folds everything all the way
-- `za`: open a fold your cursor is on
-- `zA`: open a fold your cursor is on recursively
+- `zo`: open a fold your cursor is on
+- `zO`: open a fold your cursor is on recursively
 - `zc`: close a fold your cursor is on
 - `zC`: close a fold your cursor is on recursively
+- `za`: toggle a fold your cursor is on
```

Comment:
I agree this is an improvement, but while we are editing this we should be clarified how this is different than `zo`/`zO` in that the 'a' variants opens **or closes** a fold.